app/assets/stylesheets/article-show.scss

```diff
@@ -202,7 +202,7 @@ article {
 
       &.medium {
         font-size: calc(1.85vw + 25px);
-
+        word-break : break-word;
```

Comment:
this should also appear in the other categories of titles, in my case if I add `word-break: break-word` in the `&.longest` it works. You should probably just add it to `h1` up a level. Beware of the space between `word-break` and `break-word`

Reply:
As suggested, I moved the style to the h1 and now it applies to all type of title. 

Didn't knew different classes were added depending upon the title length :)